src/Engine.sol

```diff
@@ -113,6 +115,15 @@ contract Engine is IEngine, EIP712, EIP7412 {
         ORACLE = IPyth(_oracle);
     }
 
+    /*//////////////////////////////////////////////////////////////
+                           TRUSTED FORWARDER
+    //////////////////////////////////////////////////////////////*/
+
+    /// @inheritdoc ERC2771Context
+    function trustedForwarder() public view override returns (address) {
```

Comment:
**Note:** I don't think this code is doing anything and it can be safely removed.